bigcache.go

```diff
@@ -111,7 +111,9 @@ func newBigCache(ctx context.Context, config Config, clock clock) (*BigCache, er
 			for {
 				select {
 				case <-ctx.Done():
-					fmt.Println("ctx done, shutting down bigcache cleanup routine")
```

Comment:
TBH this can be absolutely dropped, WDYT @janisz ?

Reply:
I agree